docs/source/configuration/configuration-run.rst

```diff
@@ -307,6 +311,10 @@ Classification and Regression Task
     - Yaml: ``imbalance_class_weights: 0.1,0.2,0.3``
     - Argument: ``--imbalance-class-weights 0.1,0.2,0.3``
     - Default value: ``None``
+- **return_proba**: For classification task, this configuraiton determines whether to return prediction probability or the argmax prediction. Set `true`` to return predictions probability and `false` to return the argmax prediction.
```

Comment:
typo: configuration. Can we say class prediction using argmax instead of argmax prediction?

Reply:
I rewrote with the terms extracted from scklearn's documentation.

Reply:
LGTM. please correct "configuraiton" into "configuration"

Reply:
fixed. thanks